server/services/snip.js

```diff
@@ -109,16 +110,28 @@ const validateUrl = url => {
     return true;
 }
 
+const getAuthToken = async (url, unlockPass) => {
+    const snip = await findByUrl(url);
```

Comment:
What if the snip is not there?
Need to handle the error.

Reply:
error is thrown from findByUrl